openpype/plugins/load/delete_old_versions.py

```diff
@@ -185,6 +184,14 @@ def delete_only_repre_files(self, dir_paths, file_paths, delete=True):
         return size
 
     def message(self, text):
+        from qtpy import QtWidgets, QtCore
+
+        if not QtWidgets.QApplication.instance():
+            self.log.debug(
+                "No QApplication found to show text with:\n{}" + text
+            )
+            return
```

Comment:
Considering you've moved the imports here, you wanted to avoid import error.
```suggestion
        qt_app_running = False
        try:
            from qtpy import QtWidgets, QtCore

            qt_app_running = QtWidgets.QApplication.instance()
        except Exception:
                pass

        if not qt_app_running:
            self.log.debug(
                "No QApplication found to show text with:\n{}" + text
            )
            return
```